pom.xml

```diff
@@ -166,6 +166,21 @@ THE SOFTWARE.
             <version>2.3</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.hamcrest</groupId>
+            <artifactId>hamcrest-library</artifactId>
+            <version>1.2.1</version>
```

Comment:
If needed should be a test dependency.
But generally it should be already coming from parent POM.

Reply:
Done! The first build failed, and was solved by adding those dependencies as suggested [in the wiki](https://wiki.jenkins.io/display/JENKINS/Unit+Test#UnitTest-pom.xml) (which I just fixed to include the scope as well).

I will read about Jelly forms to open a different PR updating them if that's OK.